game/src/js/scenes/DungeonScene.js

```diff
@@ -82,7 +82,7 @@ export default class DungeonScene extends BaseScene {
       cancelButtonSelector: '.exit',
       onAccept: (modal) => {
         modal.close();
-        this.spawn(0, 0);
+        this.create();
```

Comment:
I don't think this would work, since this method is called by Phaser internally.